OpenRA.Mods.Common/Activities/GuardTargetActivity.cs

```diff
@@ -0,0 +1,46 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
```

Comment:
I think something went wrong with updating this.

Reply:
Git did it right, I missed out the newly added file.